### ISSUE TITLE
DataView: Button aria-labels missing

### ISSUE THREAD

**Issue by ethanolree** [primefaces/primeng#14791]
### Describe the bug

Accessibility documentation for the DataView incorreclty claims aria properties for the list / grid view buttons are set using the locale API. There are no aria-labels for the buttons

![image](https://github.com/primefaces/primeng/assets/112996028/dcd68bc9-1856-41df-a7e6-f5f63ae8038a)

![image](https://github.com/primefaces/primeng/assets/112996028/1dc7ebb5-029c-4b72-ba25-a8414e359a63)

### Environment

N/A

### Angular version

17.0.5

### PrimeNG version

17.6.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18

### Expected behavior

Buttons have aria labels
